app/controllers/users_controller.rb

```diff
@@ -0,0 +1,5 @@
+class UsersController < ApplicationController
+  def show
+    @user = User.find(params[:id])
+  end
+end
```

Comment:
- [ ] It appears that you might have missed adding the `#index` action in your `UsersController`. Make sure to include this action to handle requests related to listing or retrieving multiple user records. It's a crucial part of your controller for managing user data. Keep up the good work! 📋👩‍💻